### PR TITLE
Create MyFile.txt

### PR DESCRIPTION
**Pull request by Sajin6k**

### CODE REVIEW

**Comment by Sajin6k** [comment]
Plz approve, if this is correct

File: MyFile.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+My new text file
```
Comment (Sajin5k) [inline MyFile.txt:1]: Change the logic

Comment (Sajin6k) [reply MyFile.txt:1]: Ok, will do that.

**Comment by Sajin5k** [comment]
This looks good.

**Pull request merged**